cobigen-templates/templates-devon4j/src/main/templates/crud_java_server_app/templates.xml

```diff
@@ -53,7 +55,9 @@
     </increment>-->
     <increment name="dataaccess_infrastructure" description="Entity infrastructure">
       <templateRef ref="ApplicationPersistenceEntity.java"/>
+      <templateRef ref="ApplicationComposedKeyPersistenceEntity.java"/>
       <templateRef ref="ApplicationEntity.java"/>
+      <templateRef ref="ApplicationComposedKeyEntity.java"  />
```

Comment:
Please create a new increment like `Entity infrastructure (Composite Key)` and remove here  `ApplicationComposedKeyPersistenceEntity.java` and `ApplicationComposedKeyEntity.java`

Reply:
Comment incorporated. Re-review needed.